src/System.Reflection.Metadata/src/System/Reflection/Metadata/BlobBuilder.cs

```diff
@@ -1090,8 +1090,8 @@ public void WriteConstant(object value)
 
         internal string GetDebuggerDisplay()
         {
-            return IsHead ? 
-                string.Join("->", GetChunks().Select(chunk => $"[{Display(chunk._buffer, chunk.Length)}]")) : 
+            return IsHead ?
+                string.Join("->", GetChunks().Select(chunk => $"[{Display(chunk._buffer, chunk.Length)}]")) :
                 $"<{Display(_buffer, Length)}>";
```

Comment:
Nit: more whitespace only diffs. I'd prefer formatting changes to be separate commits.